db/dcrsqlite/apisource.go

```diff
@@ -1047,11 +1047,14 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 
 	txs, err := db.client.SearchRawTransactionsVerbose(addr,
 		int(offset), int(count), true, true, nil)
-	if err != nil {
+	if err.Error() == "-32603: No Txns available" {
 		log.Warnf("GetAddressTransactionsRaw failed for address %s: %v", addr, err)
 		return &explorer.AddressInfo{
 			Address: address,
 		}
+	} else if err != nil {
+		log.Warnf("GetAddressTransactionsRaw failed for address %s: %v", addr, err)
+		return nil
```

Comment:
`return err`

Reply:
(this is a legit malfunction in `SearchRawTransactionsVerbose` for an address that was already validated, so we don't want to continue on with a value for `txs` that is almost certainly `nil`)

Reply:
But the return type for GetExplorerAddress() is *explorer.AddressInfo

Reply:
Ah yeah, sorry.  Nevermind.